purchase_invoice.py

```diff
@@ -176,8 +176,7 @@ def search_item(self, supplier, check_dup=True):
         texts = ['Neuen Artikel anlegen']
         texts += [i[1]['item_code'] + ' ' + i[1]['item_name'] for i in top_items]
         title = "Artikel wählen"
-        msg = "Artikel in Rechnung:\n{0}\n\nCode Lieferant: {1}\n\n".format(self.long_description,self.item_code)
-              
+        msg = "Artikel in Rechnung:\n{0}\n\nCode Lieferant: {1}\n\n".format(self.long_description, self.item_code)
```

Comment:
all changes in this file seem to concern reformatting. Therefore, this file should not be part of the pull request.

Reply:
No, there are some important changes:
1. line 943 and line 945
2. line 952 and line 953